pyproject.toml

```diff
@@ -0,0 +1,147 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "EveryVoice"
+description = "Text-to-Speech Synthesis for the Speech Generation for Indigenous Language Education Small Teams Project"
+license = { file = "LICENSE" }
+authors = [
+  { name = "Aidan Pine", email = "Aidan.Pine@nrc-cnrc.gc.ca" },
+  { name = "Eric Joanis", email = "Eric.Joanis@nrc-cnrc.gc.ca" },
+  { name = "Marc Tessier", email = "Marc.Tessier@nrc-cnrc.gc.ca" },
+  { name = "Mengzhe Geng", email = "Mengzhe.Geng@nrc-cnrc.gc.ca" },
+  { name = "Samuel Larkin", email = "Samuel.Larkin@nrc-cnrc.gc.ca" },
+]
+maintainers = [
+  { name = "Aidan Pine", email = "Aidan.Pine@nrc-cnrc.gc.ca" },
+  { name = "Eric Joanis", email = "Eric.Joanis@nrc-cnrc.gc.ca" },
+  { name = "Marc Tessier", email = "Marc.Tessier@nrc-cnrc.gc.ca" },
+  { name = "Mengzhe Geng", email = "Mengzhe.Geng@nrc-cnrc.gc.ca" },
```

Comment:
Maintainers should be Aidan, Eric and Sam.
Authors should also include Vlad and David, I think.